app/app.go

```diff
@@ -775,6 +820,7 @@ func NewApp(
 		validatorvestingtypes.ModuleName,
 		evmutiltypes.ModuleName,
 		savingstypes.ModuleName,
+		bridgetypes.ModuleName,
```

Comment:
These `SetOrder_` methods are a bit of a footgun.

All the invariants run in crisis genesis, but here the bridge state won't have been loaded from genesis yet (as it appears later in the list). So the bridge invariants run on an empty state.

Don't think its a strict problem for bridge as empty state passes invariants, but worth fixing in case that changes.

Also applies to savings and evmutil. Do you want to move these up in the list as well?